frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -175,11 +175,18 @@ const hasSimpleReceiver = (
   return false;
 };
 
-const hasIncompleteDefaultReceiver = () => {
+const numberOfIncompleteReceivers = (): number => {
```

Comment:
Not new, just wondering why `config` isn't a func arg.

Reply:
`config` started out as a func arg but I ended up passing it to virtually every function so seemed logical to make it a global to be used throughout.